components/identities/actions/identityActions.tsx

```
@@ -319,21 +318,72 @@ const IdentityActions: FunctionComponent<IdentityActionsProps> = ({
                         />
                       )}
 
+
+                    </>)}
+
+
+                  {/* I include the fale in order to hide the button */}
+
+                  {callDataEncodedDomain[0] === "1" && false &&
+                    !isAutoRenewalEnabled ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)